src/frontend/src/lib/components/receive/ReceiveButtonWithModal.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import { modalStore } from '$lib/stores/modal.store';
+	import ReceiveButton from '$lib/components/receive/ReceiveButton.svelte';
+
+	export let openModal: (modalId: symbol) => void | Promise<void>;
+	export let isOpen: boolean;
+	export let specificModalId: symbol | undefined = undefined;
+
+	const modalId = specificModalId ?? Symbol();
```

Comment:
Given that the property is reactive, it's probably safer to make this assignement reactive as well.

Reply:
right!